tests/xharness/MakefileGenerator.cs

```diff
@@ -297,7 +297,7 @@ public static void CreateMakefile (Harness harness, IEnumerable<UnifiedTarget> u
 
 				// build/[install/]run targets for specific test projects.
 				foreach (var target in allTargets) {
-					if (!target.IsExe)
+					if (!target.IsExe || target.Name.Contains ("BCLTests"))
```

Comment:
add `,StringComparison.Ordinal`

less an issue for testing code (than the SDK) but it's faster and requires less memory - which is always nice inside a loop